programs/server/config.xml

```diff
@@ -371,16 +370,40 @@
                 server - one of LDAP server names defined in 'ldap_servers' config section above.
                         This parameter is mandatory and cannot be empty.
                 roles - section with a list of locally defined roles that will be assigned to each user retrieved from the LDAP server.
-                        If no roles are specified, user will not be able to perform any actions after authentication.
-                        If any of the listed roles is not defined locally at the time of authentication, the authenthication attempt
-                         will fail as if the provided password was incorrect.
+                        If no roles are specified here or assigned during role mapping (below), user will not be able to perform any
+                         actions after authentication.
+                role_mapping - section with LDAP search parameters and mapping rules.
```

Comment:
Maybe it's better to rename `role_mapping` into `roles_search`? Just to represent the fact that all those parameters are about searching in LDAP and to make the difference between this section and the `roles` section more clear.

Reply:
`imported_roles` maybe?